pyecharts/charts/base.py

```diff
@@ -67,52 +59,20 @@ def render(
         env: Optional[Environment] = None,
         **kwargs,
     ) -> str:
-        self._prepare_render()
-        RenderEngine(env).render_chart_to_file(
-            template_name=template_name, chart=self, path=path, **kwargs
-        )
-        return os.path.abspath(path)
+        return super()._render(path, template_name, env, **kwargs)
```

Comment:
In this case super()._render(...) should be super().render(). No need to use a protected function.

default template name, 'simple_chart.html' could be an attribute of base.py